sdks/python/apache_beam/io/gcp/datastore/v1/datastoreio_test.py

```diff
@@ -238,7 +241,7 @@ def fake_run_query(req):
       elif req == kind_stat_req:
         return kind_stat_resp
       else:
-        print kind_stat_req
+        print(kind_stat_req)
```

Comment:
We have too many print statements. We should probably convert them to logs or remove them.

Reply:
I agree, however I think that probably belongs in a follow up JIRA/PR. Do you agree?